### PR TITLE
Fix tests in FederationSupportTest

### PR DESCRIPTION
**Pull request by marcos-iov**

### CODE REVIEW

File: rskj-core/src/test/java/co/rsk/peg/FederationSupportTest.java [hunk 1]
```diff
@@ -176,71 +168,32 @@ void getFederatorPublicKeys() {
...
 
-        Assertions.assertTrue(Arrays.equals(federationSupport.getFederatorBtcPublicKey(0), btcKey0.getPubKey()));
-        Assertions.assertTrue(Arrays.equals(federationSupport.getFederatorBtcPublicKey(1), btcKey1.getPubKey()));
+        assertArrayEquals(federationSupport.getActiveFederatorPublicKeyOfType(0, KeyType.BTC), btcKey0.getPubKey());
+        assertArrayEquals(federationSupport.getActiveFederatorPublicKeyOfType(1, KeyType.BTC), btcKey1.getPubKey());
 
-        Assertions.assertTrue(Arrays.equals(federationSupport.getFederatorPublicKeyOfType(0, FederationMember.KeyType.RSK), rskKey0.getPubKey(true)));
-        Assertions.assertTrue(Arrays.equals(federationSupport.getFederatorPublicKeyOfType(1, FederationMember.KeyType.RSK), rskKey1.getPubKey(true)));
+        assertArrayEquals(federationSupport.getActiveFederatorBtcPublicKey(0), btcKey0.getPubKey());
+        assertArrayEquals(federationSupport.getActiveFederatorBtcPublicKey(1), btcKey1.getPubKey());
```
Comment (julia-zack) [inline rskj-core/src/test/java/co/rsk/peg/FederationSupportTest.java:186]: why did you remove the rsk key check?

Comment (julia-zack) [inline rskj-core/src/test/java/co/rsk/peg/FederationSupportTest.java:186]: super nit, but i would revert the order. So we have the checks for BTC, RSK and MST key one after the other

**Review by julia-zack — APPROVED** [review]
Approved! Just left one super nit minor comment

File: rskj-core/src/test/java/co/rsk/peg/FederationSupportTest.java (hunk 1 above)
Comment (marcos-iov) [reply rskj-core/src/test/java/co/rsk/peg/FederationSupportTest.java:186]: Not sure I follow, that's how it is now
